pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -183,16 +186,22 @@ func (m *manager) Start(activePods ActivePodsFunc, sourcesReady config.SourcesRe
 
 	stateImpl, err := state.NewCheckpointState(m.stateFileDirectory, cpuManagerStateFileName, m.policy.Name(), initialContainers)
 	if err != nil {
-		klog.Errorf("[cpumanager] could not initialize checkpoint manager: %v\n", err)
-		panic("[cpumanager] - please drain node and remove policy state file")
+		klog.Errorf("[cpumanager] could not initialize checkpoint manager: %v, please drain node and remove policy state file", err)
```

Comment:
Logging here is redundant because the same information is printed when the Container manager errors out at the next level based on the returned error.

I'd say it's OK for now though. Let's get this PR pushed through and then come back with a cleanup PR for these kind of things afterwards-

Reply:
OK, Thanks. 👍